docs/framework/data/adonet/side-by-side-execution.md

```diff
@@ -42,7 +42,7 @@ Side-by-side execution in the .NET Framework is the ability to execute an applic
   
  In .NET Framework version 1.0, <xref:System.Data.SqlClient.SqlCommand.ExecuteReader%2A> executed all commands in the context of the **sp_executesql** stored procedure. As a result, commands that affect the state of the connection (for example, SET NOCOUNT ON), only apply to the execution of the current command. The state of the connection is not modified for any subsequent commands executed while the connection is open.  
```

Comment:
You could consider deleting the 1.0 paragraph and removing mention of 1.1-and-later from the following paragraph.